src/reformat_libyuv.c

```diff
@@ -63,6 +63,7 @@ unsigned int avifLibYUVVersion(void)
 
 // These defines are used to create a NULL reference to libyuv functions that
 // did not exist prior to a particular version of libyuv.
+// Versions prior to 1750 are considered too old and not checked.
```

Comment:
This version should match the minimum version of libyuv in CMakeLists.txt, currently 1755:

```
find_package(libyuv QUIET) # not required
if(libyuv_FOUND)
    ...
    if(NOT LIBYUV_VERSION)
        message(STATUS "libavif: libyuv found, but version unknown; libyuv-based fast paths disabled.")
    elseif(LIBYUV_VERSION LESS 1755)
        message(STATUS "libavif: libyuv (${LIBYUV_VERSION}) found, but is too old; libyuv-based fast paths disabled.")
    else()
```

Reply:
Done at https://github.com/AOMediaCodec/libavif/pull/1081